invoice/src/test/java/org/killbill/billing/invoice/tree/TestNodeInterval.java

```diff
@@ -60,8 +59,8 @@ public UUID getId() {
         public ItemsNodeInterval[] split(final LocalDate splitDate) {
             Preconditions.checkState(splitDate.compareTo(start) > 0 && splitDate.compareTo(end) < 0,
                                      String.format("Unexpected item split with startDate='%s' and endDate='%s'", start, end));
-            Preconditions.checkState(leftChild == null);
-            Preconditions.checkState(rightSibling == null);
+            Preconditions.checkState(leftChild == null, "leftChild is not null");
```

Comment:
Weird to have preconditions in tests 🤔 Maybe this should be an assertion instead?